rmw_zenoh_cpp/src/impl/pubsub_impl.cpp

```diff
@@ -22,8 +26,10 @@ void rmw_subscription_data_t::zn_sub_callback(const zn_sample * sample) {
   // Vector to store the byte array (so we have a copyable type instead of a pointer)
   std::vector<unsigned char> byte_vec(sample->value.val, sample->value.val + sample->value.len);
 
+  // Fill the static message map with the latest received message
+  //
+  // NOTE(CH3): This means that the queue size for each topic is ONE for now!!
```

Comment:
It's worth logging when a message is clobbered, just so we know that's what happened.

Reply:
Updated: ba06e8e1c13f1084a68cb5af1f9ee91dd3c31317